src/components/sidebar/Sidebar.js

```diff
@@ -16,6 +17,21 @@ export default function Sidebar({
 }) {
   const drawerList = ['Home', 'Profile', 'Subscriptions'];
 
+  const collapses = {
+    'My calendars': {
+      checkboxes: ['Birthdays', 'Reminders', 'Tasks'],
+      services: {},
+    },
+    'Other calendars': {
+      checkboxes: ['Holidays in United States'],
+      services: {
```

Comment:
I am thinking that you may expand this object like below so that all variables are contained in one space.
`
services: {
  subscribe: {
    text: "Subscribe to calendar",
    action: handlerMethod()
  }
  ...
`